docs/xks/operator-guide/kubernetes/aks.md

```diff
@@ -62,6 +62,51 @@ az aks nodepool delete --cluster-name aks-dev-we-aks1 --resource-group rg-dev-we
 
 For additional information about updating the system nodes refer to [this blog post](https://pumpingco.de/blog/modify-aks-default-node-pool-in-terraform-without-redeploying-the-cluster/).
 
+## Update AKS
```

Comment:
Please try to ensure headings are comprehensible without context (because links, snippets, search). In this case: what sort of updating are we talking about? Also, the content of this section is not actually about updates, so maybe a different heading might fit better?